src/host/sdl_host.c

```diff
@@ -20,6 +20,10 @@ DEFINE_OPTION_INT(latency, 50, "Preferred audio latency in ms");
 #define VIDEO_DEFAULT_HEIGHT 480
 #define INPUT_MAX_CONTROLLERS 4
 
+DEFINE_PERSISTENT_OPTION_INT(v_width, VIDEO_DEFAULT_WIDTH, "Video Width");
```

Comment:
Is it useful to have an overridable default window width / height?

I understand an internal resolution setting, but it seems otherwise you're likely to just use the emulator like most other applications (e.g. use the default size, resize it a bit if you need to, but normally just maximize / fullscreen the app).